### PR TITLE
Update Android Gradle Build Tool to 2.2.2

### PR DESCRIPTION
**Pull request by alexstyl**
Updates the Android Gradle build Tool to [version 2.2.2](http://tools.android.com/recent/androidstudio222isnowavailable)

**Fixed Issues**
- Instant Run bug fixes
- Gradle Plug-in Bug fixes
- 32-bit JDK for use on 32-bit Windows

**New Features**
- Android API 25 Support - Basic support for API 25 inside Android Studio to support the [Android 7.1 developer preview](https://android-developers.blogspot.com/2016/10/android71-dev-preview-available.html). We know there are still a few more updates inside Android Studio before we set API 25 as the default.
- Expanded reporting when reporting Instant Run issues - When reporting an instant run issue we now collect 

More info:
http://tools.android.com/recent/androidstudio222isnowavailable

### CODE REVIEW

**Comment by mr-archano** [comment]
_out of curiosity_: why such outdated version of Gradle?

**Comment by alexstyl** [comment]
@mr-archano Which is the latest version that can be used with Android? I just used the version that AS recommended me to use.

**Comment by mr-archano** [comment]
I'm not fully aware if there is any problem with 3.x, but I can tell you for sure 2.14.1 works great Where did you get this hint in the IDE? Are you sure you're not confusing the Android Gradle plugin with the Gradle distribution?

**Comment by devisnik** [comment]
You might want to update the PR description. The relevant changes might be found here: https://developer.android.com/studio/releases/gradle-plugin.html

**Comment by alexstyl** [comment]
You are right guys. Description and title updated 👍

File: FacesofLondon/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```
Comment (eduardb) [inline FacesofLondon/build.gradle]: `2.2.2` is out meanwhile :trollface:

**Comment by alexstyl** [comment]
@eduardb @mr-archano @devisnik I believe the PR is good for shipping

**Pull request merged**